src/SonarScanner.MSBuild.Common/AnalysisProperties/FilePropertyProvider.cs

```diff
@@ -32,9 +33,9 @@ public class FilePropertyProvider : IAnalysisPropertyProvider
     {
         private const string DescriptorId = "properties.file.argument";
         public const string DefaultFileName = "SonarQube.Analysis.xml";
-        public const string Prefix = "/s:";
+        public static readonly string[] Prefix = new[] { "s:" };
```

Comment:
FYI the change of data type in this line is what is causing the IT failures: "Prefix" is used elsewhere by code that assumes it is a single string.

Reply:
Yes i saw it, i reverted to constant, it passes now.